.gitignore

```diff
@@ -1,6 +1,7 @@
 ### Ancient Beast ###
 # generated files
 deploy/
+src/manifest.js
```

Comment:
Your next commit moved this, so this ignore is invalid.

Reply:
No it's still there, however I moved the script to generate it to the root directory since it's not really part of the game. It's like placing a gulp file in the src directory. You don't to that either

Reply:
Ahhh yes, I did misread those 2 names. Agreed.